.github/workflows/build-wheels.yml

```diff
@@ -56,14 +61,27 @@ jobs:
         with:
           platforms: all
 
-      - name: Install cibuildwheel
-        run: python -m pip install cibuildwheel==2.16.2
-
       - name: Build wheels
-        run: |
-          python -m cibuildwheel --output-dir wheelhouse statsmodels
+        uses: pypa/cibuildwheel@v2.20.0
```

Comment:
Should you just leave this as @2.20 so that we get further 2.20.# releases?

Reply:
Interesting – I didn't know that was possible, thanks! TIL 😁

Reply:
Addressed in 70a67541b17f9c0ad63ac3a5d3d174343ce32eb6.